### ISSUE TITLE
Carteira 181 Itaú

### ISSUE THREAD

**Issue by henriquevc** [BoletoNet/boletonet#310]
Galera,

Eu estou gerando boleto do Itaú pela carteira 109 e estava atualizando a data de vencimento do boleto pela mesma carteira, mas desde o dia 17/10 o Itaú começou a atualizar o boleto para a carteira 181 quando ele está vencido. Alguém está sabendo disso? Pois a dll não deixa eu gerar boleto pela carteira 181 e nem sei como implementar isso.

**Comment by fabiovank** [comment]
Opa!
Você está atualizando pelo site do Itaú, correto?
É isso mesmo, eles geram um boleto provisório, porém não altera nada pra gente, os dados de retorno vão ser do boleto original, ou seja, carteira 109.

**Comment by henriquevc** [comment]
correto, mas esqueci de avisar que quando os meus clientes vão pagar o boleto atualizado pelo boleto net na carteira 109 o sistema do itaú fala: FAVOR DIRIGIR-SE A AGENCIA. RECEBIMENTO NÃO DISPONÍVEL.

**Comment by fabiovank** [comment]
Hummm
Você troca o boleto "impresso" deles? Estranho!
Você envia um arquivo de remessa pra atualizar?

**Comment by henriquevc** [comment]
Sim, quando está vencido, quando eles clicam no link, eu atualizo a data de vencimento para a data atual e atualizo o valor com o juros mas não envio nenhum arquivo de remessa. Funcionava perfeitamente, mas depois de uns dias pra ca parou

**Comment by fabiovank** [comment]
Acredito que se você não fizer a atualização pelo site do banco, terá que enviar uma remessa de atualização desses boletos.

**Comment by pennellarodrigo** [comment]
@henriquevc conseguiu uma solução? estou enfrentando o mesmo problema...

**Comment by henriquevc** [comment]
@pennellarodrigo não consegui, nem mexi mais nisso pq não encontrei nenhuma solução, mando o boleto vencido no site de atualização do itau mesmo.

**Comment by AdrianoGeraldeli** [comment]
alguém conseguiu uma solução ?

**Comment by orejano1** [comment]
Por Favor, alguém conseguiu algum resultado positivo quanto a carteira 181? estou com o mesmo problema e ligo no Itau ninguém sabe responder se tem um parâmetro para ser alterado.

**Comment by pennellarodrigo** [comment]
Não obtive solução. Atualmente apenas alterando diretamente no site do itau
